### DISCUSSION TITLE
Feature wish - display installation location

### DISCUSSION THREAD

**Discussion by dahadd** [revk/xESP32-Faikout#585] [Ideas]
feature request: it would be nice if you could enter the installation location (e.g. living room) on the WEBUI and that this location is then displayed. see screenshot
![feature_wish](https://github.com/user-attachments/assets/f37509cf-c934-4af2-a109-dbfe25b3a38e)

**Comment by revk** [comment]
Err, wow, I mean, Ok, maybe...

Most people just name the device "LivingRoom" - why not just do that?

Like this for my Guest room AC

<img width="553" alt="Screenshot 2024-12-21 at 14 15 30" src="https://github.com/user-attachments/assets/3c57639f-df34-4269-9fc0-ec815b7ca62f" />

**Comment by dahadd** [comment]
Where can you enter this?
That is already the solution 😉

**Reply by revk** [reply]
just set the name of the device in the settings?

<img width="189" alt="Screenshot 2024-12-21 at 14 39 59" src="https://github.com/user-attachments/assets/92e16404-48fe-43ce-924d-db78d93eadb2" />

**Reply by dahadd** [reply]
Ahhhh, okay...i try

**Reply by dahadd** [reply]
Thats the solution. Thx 😉
